### ISSUE TITLE
Facilitate reading the guides by providing “next” links

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#2869]
When users get to the bottom of a guide page (like [Mission](http://canjs.com/doc/guides/mission.html)), it would be helpful to provide a link to the next page we think they should read.

**Comment by jamiemccue** [comment]
@chasenlehara What about linking to the next page in the navigation? 

Example: Mission would link to Technical Highlights

**Comment by chasenlehara** [comment]
Yeah, exactly. The goal is to guide people where we want them to go next.
